src/cmd/create.js

```diff
@@ -0,0 +1,133 @@
+/* @flow */
+import path from 'path';
+import readline from 'readline';
+import tty from 'tty';
+
+import {fs} from 'mz';
+import mkdirp from 'mkdirp';
+import promisify from 'es6-promisify';
+
+import {createLogger} from '../util/logger';
+import {UsageError, isErrorWithCode} from '../errors';
+
+const log = createLogger(__filename);
+const defaultAsyncMkdirp = promisify(mkdirp);
+
+export type CreateParams = {|
```

Comment:
we should split these properties into two parameters and flow types like we do for the other commands:
- the first one should be named `SomethingParams` (e.g. `CreateCmdParams`) and it should only contains what is supposed to be in the regular parameters of the command (e.g. `dirPath` in this case)
- the second one should be named `SomethingOptions` (e.g. `CreateCmdOptions`) and it should only contains the optional injected dependencies (`stdin?` in this case), which are used mostly to override the dependencies in the unit tests